packages/invoices/src/OutboundShipment/DetailView/SidePanel/PricingSection.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { memo } from 'react';
+import {
+  formatTax,
+  Tooltip,
+  Grid,
+  DetailPanelSection,
+  PanelField,
+  PanelLabel,
+  PanelRow,
+  useTranslation,
+  useToggle,
+  IconButton,
+  useFormatCurrency,
+  PricingNode,
+  NumUtils,
+  InfoIcon,
+  MenuDotsIcon,
+} from '@openmsupply-client/common';
+import { useOutboundFields } from '../../api';
+import { OutboundServiceLineEdit } from '../OutboundServiceLineEdit';
+import { TaxEdit } from '../modals';
+
+type PricingGroupProps = {
+  pricing: PricingNode;
+};
+
+const InfoTooltip = ({ title }: { title: string }) => (
+  <Tooltip title={title}>
+    <div style={{ transform: 'scale(0.7)' }}>
+      <InfoIcon fontSize="small" />
+    </div>
+  </Tooltip>
+);
```

Comment:
Maybe this should be a more generic component to use if it's helpful. The transform is a hacky 😁

Reply:
I was wondering about the size of the icon! 
Agree - this one could be moved up into common, it would be helpful to reuse